docs/book/development.md

```diff
@@ -0,0 +1,129 @@
+# vSphere CSI Driver - Driver Development
+
+## How to build this project
+
+- Fork the project on [github](https://github.com/kubernetes-sigs/vsphere-csi-driver)
+
+- Clone the project
+
+``` sh
+git clone https://github.com/your_github_user_name/vsphere-csi-driver.git
+
+```
+
+- Create a branch to the issues you are working on
+
+``` sh
+git checkout -b my_fix
+```
+
+- Make code changes for csi driver
+
+- Build csi driver
+
+``` sh
+make build
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags '-extldflags "-static" -w -s -X "sigs.k8s.io/vsphere-csi-driver/pkg/csi/service.version=73865e82-dirty"' -o /Users/lipingx/go/src/gitlab.eng.vmware.com/hatchway/vsphere-csi-driver/.build/bin/vsphere-csi.linux_amd64 cmd/vsphere-csi/main.go
+go: finding k8s.io/sample-controller v0.0.0-00010101000000-000000000000
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -ldflags '-extldflags "-static" -w -s' -o /Users/lipingx/go/src/gitlab.eng.vmware.com/hatchway/vsphere-csi-driver/.build/bin/syncer.linux_amd64 cmd/syncer/main.go
+go: finding k8s.io/sample-controller v0.0.0-00010101000000-000000000000
+```
+
+- Run required static check on the code
```

Comment:
Can we re-word this. `make check` also executes `fmt lint mdlint shellcheck staticcheck vet`. not just staticcheck?

Reply:
Fixed.